Mathlib/Util/CountHeartbeats.lean

```diff
@@ -176,3 +174,55 @@ elab "count_heartbeats! " n:(num)? "in" ppLine cmd:command : command => do
 end CountHeartbeats
 
 end Mathlib
+
+/-!
+#  The "countHeartbeats" linter
+
+The "countHeartbeats" linter counts the hearbeats of every declaration.
+-/
+
+open Lean Elab Command
+
+namespace Mathlib.Linter
+
+/--
+The "countHeartbeats" linter counts the hearbeats of every declaration.
+
+The effect of the linter is similar to `#count_hearbeats in xxx`, except that it applies
```

Comment:
```suggestion
The effect of the linter is similar to `count_hearbeats in xxx`, except that it applies
```

Reply:
Thanks!  I was really convinced that there was a `#` in the other command.  Also, fixing this, I noticed an extra spelling mistake in that command!  :facepalm: